ui/admin/app/components/form/host-catalog/aws/index.hbs

```diff
@@ -139,8 +137,7 @@
     @value={{@model.access_key_id}}
     @label={{t 'form.access_key_id.label'}}
     @helperText={{t 'form.access_key_id.help'}}
-    @linkText={{t 'form.access_key_id.link-text'}}
-    @link='https://learn.hashicorp.com/tutorials/boundary/cloud-host-catalogs#set-up-cloud-hosts'
```

Comment:
What about the links?

Reply:
the links will come in a separate PR as I understand from the comment on slack that it needs to be added on the boundary repo first?

Reply:
It doesn't matter the order really as long as they're released at the same time.

Reply:
added a separate pr for the [doc-links]( https://github.com/hashicorp/boundary-ui/pull/970  )